src/content/docs/kubernetes-pixie/kubernetes-integration/understand-use-data/find-use-your-kubernetes-data.mdx

```diff
@@ -2141,6 +2163,16 @@ Query the `K8sContainerSample` event for container data:
         Total size of memory mapped files used by this container, in bytes.
       </td>
     </tr>
+
+    <tr>
+      <td>
+        `containerOOMEventsDelta`
+      </td>
+
+      <td>
+        Number of out of memory events observed for the container (requires k8s v1.24+).
```

Comment:
```suggestion
        Number of "out of memory" events observed for the container (requires k8s v1.24+).
```
Quoting to group them together, I think it is clearer but I'm not sure if it is correct

Reply:
Done in 0b782eae3fd7da1856370e2efed260a4fe70f300.